ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/fees/BaseFee.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core.fees;
+
+public class BaseFee {
+  private static final long BOUNDING_DENOMINATOR = 8;
```

Comment:
Will this ever change across hard forks or are the designers fairly sure it is static?

Reply:
Actually the value is defined in `FeeMarket` interface: `getBasefeeMaxChangeDenominator`
Removed the constant in favor of the interface method